### ISSUE TITLE
Check full accessibility for color blindness

### ISSUE THREAD

**Issue by maxsivry** [Rapa-Nui-Language-Preservation-Project/rapa-nui-website#24]
Currently its not up to accessibility standards, and I also think the icons should change to be more appealing. Maybe this means that icons will have a blurred background to contrast more against the map, or change colors.

Up to your discretion on what icons to use/ colors, please send to the groupchat different mockups and we can decide together

**Comment by maxsivry** [comment]
to make sure the site is up to standard for color contrast should be straightforward, you can see here how to run an audit with lighthouse in dev tools:
https://developer.chrome.com/docs/devtools/accessibility/reference#overview


making sure a screen reader / keyboard can navigate is a bit more involve imo but nothing too crazy. 
https://web.dev/articles/semantics-builtin#screen-readers
